README.md

```diff
@@ -7,6 +7,28 @@ If one wishes to use [GDAL's VRT functionality](https://www.gdal.org/gdal_vrttut
 The code in this repository attempts to address that issue by wrapping GDAL datasets in objects which abstract one or more identical datasets; the wrapped datasets can be safely used from multiple threads with less contention than would be the case with a simple mutex around one GDAL dataset.
 APIs are provided for C and Java.
 
+# Installation #
+
+These bindings require a GDAL installation on your machine with the appropriate matching version:
+
+| GDAL Warp Bindings | GDAL  | Shared Library {so|dylib|dll} |
+|--------------------|-------|-------------------------------|
+| 1.1.x              | 3.1.2 | libgdal.27                    |
```

Comment:
ouch it looks broken for me (visually; https://github.com/geotrellis/gdal-warp-bindings/tree/69da5bc7b5d74d07e45292220f9c5a9c2652329a#installation); 

let's include OS versions into that table as well? I.e. Windows is a little bit behind usually.